neo/Wallets/VerificationContract.py

```diff
@@ -0,0 +1,114 @@
+from neo.SmartContract.Contract import Contract
+from neo.Cryptography.Crypto import Crypto
+from neo.SmartContract.ContractParameterType import ContractParameterType
+from neo.VM.OpCode import CHECKSIG
+from neo.IO.Helper import Helper as IOHelper
+
+
+class VerificationContract(Contract):
+    def __init__(self, redeem_script=None, param_list=None, pubkey_hash=None):
+        super(VerificationContract, self).__init__(
+            redeem_script=redeem_script,
+            param_list=param_list,
+            pubkey_hash=pubkey_hash
+        )
+
+    @property
+    def IsStandard(self):
```

Comment:
#overwriting Contract.IsStandard because Script should not need to be unhexlified